travis/publish.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+# Setup login
+openssl aes-256-cbc -K $encrypted_fb15e83f0f9f_key -iv $encrypted_fb15e83f0f9f_iv \
+    -in .dockercfg.enc -out ~/.dockercfg -d
+
+
+JDK="oraclejdk7"   # what JDK we use to build the WAR that goes in the docker image
+NAME="camptocamp/mapfish_print"
+
+function publish {
+    local version=$1
+    export DOCKER_VERSION=`echo "${version}" | sed -e 's/^release\///' | sed -e 's/\//_/g'`
+
+    echo "Deploying image to docker hub for tag ${DOCKER_VERSION}"
+    ./gradlew --console=plain createDocker
+    docker push "${NAME}:${DOCKER_VERSION}"
+}
+
+if [ "${TRAVIS_JDK_VERSION}" != "${JDK}" ]
+then
+    echo "Only publishing with ${JDK}"
+    exit 0
+fi
+
+if [ "${TRAVIS_PULL_REQUEST}" == "true" ]
+then
+    echo "Not deploying image for pull requests"
+    exit 0
+fi
+
+if [ "${TRAVIS_BRANCH}" == "development" ] || [ "${TRAVIS_BRANCH}" == "docker" ]
+then
+  publish latest
```

Comment:
Every time a PR is merged, you want to update the Docker image with the tag "latest"? And when a (Git) tag is built, you create a new image with the tag name?

You probably are using "latest" to avoid that too many images are created? The disadvantage is that you don't know which version/revision of MFP was used for the Docker image "latest".